integrations/adobe-analytics/lib/chromecast-heartbeat.js

```diff
@@ -0,0 +1,176 @@
+
+var chromecastHeartbeat = {
+
+  /**
+  * Returns a mediaMetadata Object 
+  * @param {Track} track
+  * @returns {mediaMetadata} Maps the contextValue prop if it exists in the track.properties
+  */
+  extractMediaMetadata: function (track) {
+    var props = track.properties();
+    var mediaMetadata = {};
+    if (props.contextValues) {
+      for (var customProp in props.contextValues) {
+        if (props[customProp]) {
+          mediaMetadata[props.contextValues[customProp]] = props[customProp];
+        }
+      }
+    }
+    return mediaMetadata;
+  },
+  
+  chromecastInit: function (track) {    
+    var props = track.properties();
+    var mediaMetadata = window.extractMediaMetadata(track);   
+    window.ADBMobile.config.setDebugLogging(true);
+    var qosInfoSettings = {
+      bitrate: props.bitrate | 1,
```

Comment:
Should this be logical OR `||` rather than bitwise OR? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_OR

Reply:
great catch - it should be OR.